### PR TITLE
Changes in .md files 

### PR DESCRIPTION
**Pull request by pratap-akhand**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build was run locally and any changes were pushed
- [ ] Lint has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?
Modifying the .md files and location of image files so that it can be used in mkdocs tool.

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+Ingest Couchbase
+----------------
+
+1. Couchbase cluster/single instance using (XDCR ingestion mechanism) 
+2. Couchbase backup peice using (CBBACKUPMGR ingestion mechanism) - Zero Touch Production.
+
+### <a id="requirements-plugin"></a>Prerequisites
+**Source Requirements:** Couchbase database user with following privileges
+*   XDCR_ADMIN
+*   DATA_MONITOR
```
Comment (fdrozdowski) [inline docs/docs/index.md:18]: Looks like these are not rendered as bullets. Maybe the extra spaces are the issue?

File: docs/docs/index.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+
+**Staging Requirements**: O/S user with following privileges
+1. Regular o/s user.
+2. Execute access on couchbase binaries [ chmod -R 775 /opt/couchbase ]
+3. Empty folder on host to hold delphix toolkit  [ approximate 2GB free space ]
+4. Empty folder on host to mount nfs filesystem. This is just an empty folder with no space requirements and act as base folder for nfs mounts.
+5. sudo privileges for mount, umount. See sample below assuming `delphix_os` is used as delphix user.
```
Comment (fdrozdowski) [inline docs/docs/index.md:25]: For some reason these are not rendered as an ordered list.

File: docs/docs/index.md [hunk 3]
```diff
@@ -1 +1,58 @@
...
+  
+       `/opt/couchbase/bin/couchbase-cli bucket-list --cluster <sourcehost>:8091  --username $username --password $password`
+    
+    * Create config file using below command. This file will be required at the time of dSource creation using CBBACKUPMGR.
+      
+      `/opt/couchbase/bin/cbbackupmgr config --archive /u01/couchbase_backup --repo delphix`
+    
+    * Get data from source host in backup directory of staging host
+    
+      `/opt/couchbase/bin/cbbackupmgr backup -a /u01/couchbase_backup -r delphix -c couchbase://<hostname> -u user -p password`
```
Comment (fdrozdowski) [inline docs/docs/index.md:43]: Looks like all of this is rendered as one code block.

File: docs/docs/index.md [hunk 4]
```diff
@@ -1 +1,58 @@
...
+       
+  
+
+**Target Requirements**: O/S user with following privileges
+1. Regular o/s user.
+2. Execute access on couchbase binaries [ chmod -R 775 /opt/couchbase ]
+3. Empty folder on host to hold delphix toolkit  [ approximate 2GB free space ]
+4. Empty folder on host to mount nfs filesystem. This is just and empty folder with no space requirements and act as base folder for nfs mounts.
+5. sudo privileges for mount, umount. See sample below assuming `delphix_os` is used as delphix user.
```
Comment (fdrozdowski) [inline docs/docs/index.md:52]: The ordered list is not rendered correctly. The numbers are sometimes in the same line.

File: docs/mkdocs.yml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  - 'UA-35429885-3'
+  - 'auto'
+
+repo_url: https://github.com/delphix/couchbase-plugin
+repo_name: Github
+use_directory_urls: false
+pages:
+  - Overview: 'index.md'
+  - Installation: 'Installation.md'
+  - PluginDevelopment : 'PluginDevelopment.md'
```
Comment (fdrozdowski) [inline docs/mkdocs.yml:26]: Is it possible to have a space in between the words? "Plugin Development"?

File: docs/docs/.pages [hunk 6]
```diff
@@ -1,7 +1,7 @@
 arrange:
-  - Overview.md
-  - InstallPlugin.md
-  - BuildPlugin.md
-  - Discovery.md
-  - Linking.md
-  - Provisioning.md
+  - Overview: 'index.md'
+  - [Installation]: 'Installation.md'
+  - PluginDevelopment : 'PluginDevelopment.md'
```
Comment (fdrozdowski) [inline docs/docs/.pages:4]: Ditto: can we add a space here?

File: docs/docs/PluginDevelopment.md [hunk 7]
```diff
@@ -21,24 +20,31 @@ For example:
 ```bash
     dvp build
 ```
```
Comment (fdrozdowski) [inline docs/docs/PluginDevelopment.md:22]: It looks like all these lines are rendered in one code block.

File: docs/docs/PluginDevelopment.md [hunk 8]
```diff
@@ -21,24 +20,31 @@ For example:
...
 
-### <a id="run_unit_test_case"></a>Download plugin logs
-#### Plugin Logs:
-Download the plugin logs using below command:
-
-```dvp download-logs -c plugin_config.yml -e <Delphix_Engine_Name> -u admin --password <password>```
-
-#### Unit test logs: 
 ##### SummaryReport:
```
Comment (fdrozdowski) [inline docs/docs/PluginDevelopment.md:40]: Do you think we could add an extra space -> "Summary Report"?

File: docs/docs/PluginDevelopment.md [hunk 9]
```diff
@@ -1,6 +1,5 @@
-
-Steps to build, upload and run unit tests for plugin:
------------------------------------------------------
+Build plugin
+------------
 
   1. Create a virtual environment and install required libraries(dvp, pytest, pytest-html & pytest-cov) using script `virtualEnvSetup.sh`.
```
Comment (fdrozdowski) [inline docs/docs/PluginDevelopment.md:4]: nit: add a space after "libraries"

File: docs/docs/PluginDevelopment.md [hunk 10]
```diff
@@ -21,24 +20,31 @@ For example:
 ```bash
     dvp build
 ```
-    
-   4. Upload the `artifacts.json` ( generated in step 3 ) on Delphix Engine:
+
+
+Upload plugin
+----------------
+   Upload the `artifacts.json` ( generated in step 3 ) on Delphix Engine:
```
Comment (fdrozdowski) [inline docs/docs/PluginDevelopment.md:27]: nit: I think the artifact is called `artifact.json`?

Comment (fdrozdowski) [inline docs/docs/PluginDevelopment.md:27]: nit: extra space: ( generated in step 3 ) -> (generated in step 3)

File: docs/docs/PluginDevelopment.md [hunk 11]
```diff
@@ -21,24 +20,31 @@ For example:
...
 ```bash
     dvp upload -e <Delphix_Engine_Name> -u <username> --password <password>
 ```
-   5. Unit test run: Make sure to build the source code( using `dvp build` ) before running unit tests
+
+
+Unit Test of Plugin code
+------------------------
+   Unit test run: Make sure to build the source code( using `dvp build` ) before running unit tests
```
Comment (fdrozdowski) [inline docs/docs/PluginDevelopment.md:35]: nit: extra space: `dvp build` ) -> `dvp build`)

File: docs/docs/index.md (hunk 1 above)
Comment (pratap-akhand) [reply docs/docs/index.md:18]: Fixed.

File: docs/docs/index.md (hunk 2 above)
Comment (pratap-akhand) [reply docs/docs/index.md:25]: Fixed. Yes, an extra space was required.

File: docs/docs/index.md (hunk 3 above)
Comment (pratap-akhand) [reply docs/docs/index.md:43]: Fixed.

File: docs/docs/index.md (hunk 4 above)
Comment (pratap-akhand) [reply docs/docs/index.md:52]: Fixed.

File: docs/mkdocs.yml (hunk 5 above)
Comment (pratap-akhand) [reply docs/mkdocs.yml:26]: Yes, adding extra space in `PluginDevelopment`

File: docs/docs/.pages (hunk 6 above)
Comment (pratap-akhand) [reply docs/docs/.pages:4]: Fixed

File: docs/docs/PluginDevelopment.md (hunk 7 above)
Comment (pratap-akhand) [reply docs/docs/PluginDevelopment.md:22]: Fixed.

File: docs/docs/PluginDevelopment.md (hunk 8 above)
Comment (pratap-akhand) [reply docs/docs/PluginDevelopment.md:40]: Fixed

File: docs/docs/PluginDevelopment.md (hunk 9 above)
Comment (pratap-akhand) [reply docs/docs/PluginDevelopment.md:4]: Fixed

File: docs/docs/PluginDevelopment.md (hunk 10 above)
Comment (pratap-akhand) [reply docs/docs/PluginDevelopment.md:27]: Fixed

Comment (pratap-akhand) [reply docs/docs/PluginDevelopment.md:27]: Fixed

File: docs/docs/PluginDevelopment.md (hunk 11 above)
Comment (pratap-akhand) [reply docs/docs/PluginDevelopment.md:35]: Fixed

**Review by jas-sumal — APPROVED** [review]

**Review by varunshrivastava2502 — APPROVED** [review]

**Pull request merged**